features/networking/egress-ip.feature

```diff
@@ -232,3 +232,127 @@ Feature: Egress IP related features
     Then the step should succeed
     And the output should not contain "reg0=0x"
     """
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-19973
+  @admin
+  @destructive
+  Scenario: The egressIP should still work fine after the node or network service restarted
+    Given the valid egress IP is added to the node
+    And I have a project
+    And I have a pod-for-ping in the project
+
+    # add the egress ip to the project
+    Given as admin I successfully merge patch resource "netnamespace/<%= project.name %>" with:
+      | {"egressIPs": ["<%= cb.valid_ip %>"]} |
+
+    When I execute on the pod:
+      | curl | -s | --connect-timeout | 5 | ifconfig.me |
+    Then the step should succeed
+    And the output should contain "<%= cb.valid_ip %>"
+
+    # Reboot the node which patched egressIP
+    Given the host is rebooted and I wait it up to 600 seconds to become available
```

Comment:
If cluster has three nodes, and egressIP is added to the node-A, line 255 is to reboot the host/node, I assume you code make sure the host is just the node-A, just double check

Reply:
The Scenario of this case is "The egressIP should still work fine after the node or network service restarted", how about add one more step to just restart network service without rebooting the node? WDYT

Reply:
Okay, will add it.

Reply:
> If cluster has three nodes, and egressIP is added to the node-A, line 255 is to reboot the host/node, I assume you code make sure the host is just the node-A, just double check

@huiran0826 Could you double check this?

Reply:
I have checked this, and it is sure use the selected node.

Reply:
Added the step for "restart network service" and local run passed. Thanks!